src/Lang/ne.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Date Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used by the date library. Each line can
+    | have a singular and plural translation separated by a '|'.
+    |
+    */
+
+    'ago'       => ':time पहिले',
+    'from_now'  => ':time देखी',
+    'after'     => ':time पछि',
+    'before'    => ':time अघि',
+    'year'      => ':count वर्ष',
+    'month'     => ':count महिना',
+    'week'      => ':count हप्ता',
+    'day'       => ':count दिन',
+    'hour'      => ':count घण्टा',
+    'minute'    => ':count मिनेट',
+    'second'    => ':count सेकेण्ड',
+
+    'january'   => 'जेनवरी',
+    'february'  => 'फेब्रुअरी',
+    'march'     => 'मार्च',
+    'april'     => 'अप्रिल',
+    'may'       => 'मे',
+    'june'      => 'जून',
+    'july'      => 'जूलाई',
+    'august'    => 'अगस्ट',
+    'september' => 'सेप्टेम्बर',
+    'october'   => 'अक्टोबर',
+    'november'  => 'नोभेम्बर',
+    'december'  => 'डिसेम्बर',
+
+    'sunday'    => 'आइतवार',
```

Comment:
Not वार it's बार :)

Reply:
Actually this should be आईतबार

Reply:
sure. might have to send a PR to nepalipatro.com.np guys! :)

For refrence: http://pustakalaya.org/view.php?pid=Pustakalaya:2943 (Nepali Dictionary)

Reply:
@ankitpokhrel If you look at the dictionary: http://pustakalaya.org/view.php?pid=Pustakalaya:2943

It says अाइतबार (Page 94, entry: अाइत)